js/spProcessMutation.js

```diff
@@ -0,0 +1,135 @@
+// @ts-check
```

Comment:
Suggestion: we could make this slightly easier to use by refactoring like this:

- `util.js` -> promisification stuff, not replicache specific
- `replicache.js` -> spProcessMutation, getMutationID, setMutationID, getReplicacheStateID but *not* the actual handlers, like `createTodo`
- `mutators.js` -> `createTodo` and friends

This isn't all the way to a real framework, but I think would make the sample easier to copy/paste for users.

Reply:
Sounds good. I'll do that.